docs/pages/style-spec.js

```diff
@@ -1053,14 +1055,51 @@ export default class extends React.Component {
                                             }
                                         }`)}
                                 </div>
-                                Then the style could refer to this sprite image
-                                by creating a symbol layer with the layout
-                                property
-                                <code>"icon-image": "poi"</code>, or with the
-                                tokenized value{' '}
-                                <code>"icon-image": "{`{icon}`}"</code> and
-                                vector tile features with a <code>icon</code>{' '}
-                                property with the value <code>poi</code>.
+                                <p>
+                                    Then the style could refer to this sprite
+                                    image by creating a symbol layer with the
+                                    layout property
+                                    <code>"icon-image": "poi"</code>, or with
+                                    the tokenized value{' '}
+                                    <code>"icon-image": "{`{icon}`}"</code> and
+                                    vector tile features with a{' '}
+                                    <code>icon</code> property with the value{' '}
+                                    <code>poi</code>.
+                                </p>
+                                <p>
+                                    Apart from the required <code>width</code>,
+                                    <code>height</code>, <code>x</code>, and{' '}
+                                    <code>y</code>
```

Comment:
Do we need a space here?

```suggestion
                                    <code>y</code>{' '}
```

![image](https://user-images.githubusercontent.com/2180540/70734854-083f4080-1cdb-11ea-9216-d2bdcc704196.png)